docs/source/pipelines.mdx

```diff
@@ -0,0 +1,150 @@
+There are tags on the Model Hub that allow you to filter for a model you'd like to use for your task. 
+Once you've picked an appropriate model, load it with the `from_pretrained("{model_id}",from_transformers=True)` method for corresponding `ORTModelFor*` 
+and [`AutoTokenizer'] class. For example, load the [`ORTModelForQuestionAnswering`] class for a question answering modeling task:
+
+```py
+>>> from transformers import AutoTokenizer
+>>> from optimum.onnxruntime import ORTModelForQuestionAnswering
+>>> from optimum import optimum_pipeline
+
+>>> tokenizer = AutoTokenizer.from_pretrained("deepset/roberta-base-squad2")
+>>> model = ORTModelForQuestionAnswering.from_pretrained("deepset/roberta-base-squad2",from_transformers=True)
+
+>>> onnx_qa = optimum_pipeline("question-answering", model=model, tokenizer=tokenizer)
+>>> question = "Whats my name?"
+>>> context = "My Name is Philipp and I live in Nuremberg."
+
+>>> pred = onnx_qa(question=question, context=context)
+```
+
+### Use optimum model
+
+The [`optimum_pipeline`] is tightly integrated with [Model Hub](https://huggingface.co/models) and can load optimized models directly, e.g. Onnxruntime. 
+There are tags on the Model Hub that allow you to filter for a model you'd like to use for your task. 
+Once you've picked an appropriate model, load it with the `.from_pretrained` method for corresponding `ORTModelFor*` 
+and [`AutoTokenizer'] class. For example, load the [`ORTModelForQuestionAnswering`] class for a question answering modeling task:
+
+```py
+>>> from transformers import AutoTokenizer
+>>> from optimum.onnxruntime import ORTModelForQuestionAnswering
+>>> from optimum import optimum_pipeline
+
+>>> tokenizer = AutoTokenizer.from_pretrained("optimum/roberta-base-squad2")
+>>> model = ORTModelForQuestionAnswering.from_pretrained("optimum/roberta-base-squad2")
+
+>>> onnx_qa = optimum_pipeline("question-answering", model=model, tokenizer=tokenizer)
+>>> question = "Whats my name?"
+>>> context = "My Name is Philipp and I live in Nuremberg."
+
+>>> pred = onnx_qa(question=question, context=context)
+```
+
+
+### Optimizing and Quantizing in Pipelines
+
+The [`optimum_pipeline`] can not only run inference it also provides arguments to quantize and optimize your model on the fly.
+Once you've picked an appropriate model, load it with the `.from_pretrained` method for corresponding `ORTModelFor*` 
+and [`AutoTokenizer'] class. For example, load the [`ORTModelForQuestionAnswering`] class for a question answering modeling task and provide 
+the `do_optimization=True` and/or `do_quantization=True` arguments:
+
+```py
+>>> from transformers import AutoTokenizer
+>>> from optimum.onnxruntime import ORTModelForQuestionAnswering
+>>> from optimum import optimum_pipeline
+
+>>> tokenizer = AutoTokenizer.from_pretrained("optimum/roberta-base-squad2")
+>>> model = ORTModelForQuestionAnswering.from_pretrained("optimum/roberta-base-squad2")
+
+>>> model.optimize()
+>>> model.quantize()
```

Comment:
I think this is redundant with `do_optimization=True` and `do_quantization=True` at lines 120 and 121. I guess we want to delete these lines and keep lines 120-121 since this is an example of quantization/optimization inside the pipeline.